tests/events/apiGatewayProxyV2LambdaAuthorizerEvent.json

```diff
@@ -24,10 +24,16 @@
         "stage": "$default",
         "requestId": "id",
         "authorizer": {
```

Comment:
can we add a custom authorizer context for completion besides JWT claims/scopes?

```json
"tenant_id": "863f6cc5-1277-4509-b095-945250181cf4"
```